### ISSUE TITLE
Red tab text on account pages too low contrast in dark mode

### ISSUE THREAD

**Issue by parlough** [dart-lang/pub-dev#8306]
<img width="400" alt="Tab text red color is too low contrast in dark mode" src="https://github.com/user-attachments/assets/5266af51-adcd-43fe-bd65-539747acf14c">

---

It's also not super clear to me why these tabs are red. To indicate the contents are only accessible if logged in? Perhaps the text should be colored normally and the lock icon could be red? Too much red on the page is a bit jarring.

**Comment by sigurdm** [comment]
> To indicate the contents are only accessible if logged in? 

Yeah - I think that is the intention

**Comment by parlough** [comment]
Resolved by https://github.com/dart-lang/pub-dev/pull/8533
